css/src/tokens/palette.scss

```diff
@@ -148,6 +150,10 @@ $palette-purple-100: #03002c !default;
 $palette-purple-opacity-30: hsl(251deg 47% 18% / 30%) !default;
 $palette-purple-opacity-70: hsl(251deg 47% 18% / 70%) !default;
 
+// Pink
+
+$palette-pink: #702573 !default;
```

Comment:
We're adding pink? I must have missed this use case?

Reply:
Oh, I just realized this color is a part of the pink palette rather than purple 🤷🏻‍♀️

Reply:
Removed